pyproject.toml

```diff
@@ -90,7 +90,7 @@ pandas = {version = "^2.0.1", optional = true}
 telethon = {version = "^1.28.5", optional = true}
 zep-python = {version="^0.25", optional=true}
 chardet = {version="^5.1.0", optional=true}
-
+openlm = {version = "^0.0.5", optional = true}
```

Comment:
if this isn't added to one of the extras it'll be installed automatically. could we either delete or add to "all" (line 186)